packages/json-schemas/schemas/basic_type_schemas.ts

```diff
@@ -4,6 +4,12 @@ export const addressSchema = {
     pattern: '^0x[0-9a-f]{40}$',
 };
 
+export const hexSchema = {
+    id: '/Hex',
+    type: 'string',
+    pattern: '^0x[0-9a-f]',
```

Comment:
It should be `^0x([0-9a-f][0-9a-f])*$`
It's not caught by tests because you've changed them to test other schema